causallib/time_varying/GFormula.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+import abc
+from typing import Optional, Any, OrderedDict
+import pandas as pd
+from ..estimation.base_estimator import IndividualOutcomeEstimator
+from sklearn.base import BaseEstimator as SKLearnBaseEstimator
+from .regression_curve_fitter import RegressionCurveFitter #TODO Referenced from Survival module. Assuming we need have similar wrapper class for fitting curve
+
+
+#TODO
+class CovariateModels(OrderedDict):
```

Comment:
What is the purpose of this class?

Reply:
I wanted to coerce the type of Models for each Covariate variables. I'll just leave default OrderedDict for now.